terraform/modules/cloudfront/cloudfront.tf

```diff
@@ -122,6 +122,29 @@ resource "aws_cloudfront_distribution" "default" {
     viewer_protocol_policy = "redirect-to-https"
   }
 
+  cache_behavior {
+    allowed_methods  = ["GET", "HEAD"]
+    cached_methods   = ["GET", "HEAD"]
+    target_origin_id = "${var.project_name}-${var.environment}-default-origin"
+
+    path_pattern = "/"
```

Comment:
`/` is the root path for the service where every user arrives too. This renders the same content as a request to `/jobs`, which is used when a user searches or sorts the content. For sorting and searching we want up to date data so this behaviours timings can be higher for root, I propose.